tiledb/common/test/unit_var_length_view.cc

```diff
@@ -271,28 +271,13 @@ TEST_CASE("var_length_view: Viewness", "[var_length_view]") {
   std::vector<double> s = {1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0};
   std::vector<size_t> o = {0, 3, 6, 10};
   std::vector<size_t> m = {0, 3, 6, 10};
-  std::vector<double> q = {
```

Comment:
Why did we remove code here?

Reply:
Comparing iterators of different collections like `v.begin() != w.begin()` fails with an assert on Windows in Debug mode. I removed these asserts and the variables that became unused.

Reply:
Check with Andrew here... I want to make sure we are not removing coverage that we need.

Reply:
@lums658 thoughts?

Reply:
Per offline feedback, we should guard these asserts to not run on MSVC.